common/customizable_localization/99_fr_custom_loc.txt

```diff
@@ -1770,4 +1785,10 @@ FR_ArtifactWeaponBaseType_Le = {
 	log_loc_errors = no
 	parent = ArtifactWeaponBaseType
 	suffix = _FR_Le
+}
+
+FR_GetRandomPositiveTrait_OnA = { 
+	log_loc_errors = no
+	parent = GetRandomPositiveTrait
+	suffix = _FR_On
```

Comment:
@Gaben1025 Do you know what is going on here?
```
[pdx_persistent_reader.cpp:228]: Error: "Failed to read key reference: GetRandomPositiveTrait: GetRandomPositiveTrait, near line: 1792
" in file: "common/customizable_localization/99_fr_custom_loc.txt" near line: 1792
[jomini_custom_text.cpp:95]: Invalid/missing scope type for custom localization 'FR_GetRandomPositiveTrait_OnA', at file: 'common/customizable_localization/99_fr_custom_loc.txt', line: '1790-1794'
```

Reply:
it should be fixed now

Reply:
LGTM!